### ISSUE TITLE
Error in Quickstart terminal on Windows 10

### ISSUE THREAD

**Issue by kastl** [fnproject/cli#66]
I get the following error when running from the Docker quickstart terminal on windows 10 -

`fn.exe start`
`docker: Error response from daemon: invalid bind mount spec "C:\\Program Files\\Git/data:/app/data": invalid mode: /app/data.`

**Comment by treeder** [comment]
Are you in powershell?

**Comment by kastl** [comment]
No, just the Docker quick start terminal that gets installed with Docker on Windows. It is git bash based. This is normally where I do all my docker commands from.

**Comment by hibooboo2** [comment]
Seems to be because windows uses `\\` for file urls and everyone else uses `/` Might need conditional logic depending on windows vs unix / mac...

**Comment by denismakogon** [comment]
Maybe try powershell? Travis was referring to  that for all people who use Windows. So, maybe that is the case.
